assests/javaScript/home.js

```diff
@@ -0,0 +1,39 @@
+const movieList = document.getElementById("movie-list");
+let markUp = "";
```

Comment:
Why this variable is in global scope. you can probably move it to function scope?

Reply:
You are right!
Done